homework2/Trie.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace homework2;
+/// <summary>
+/// Contains methods for working with Trie
+/// </summary>
+public class Trie
+{
+    /// <summary>
+    /// Represents a Trie node
+    /// </summary>
+    private class TrieNode
+    {
+        public TrieNode()
+        {
+            Children = new Dictionary<char, TrieNode>();
+        }
+        public Dictionary<char, TrieNode> Children { get;}
+        public bool IsEndOfWord { get; set; }
+        public int HowManyStartsWith { get; set; }
+    }
+ 
+    public Trie() 
+    {
+        root = new TrieNode();
+        root.HowManyStartsWith = 0;
+        Size = 0;
+    }
+    
+    private TrieNode root;
+    
+    public int Size { get; private set; }
+    
+    /// <summary>
+    /// Adds element to Trie
+    /// </summary>
+ 
+    public bool Add(string element)
```

Comment:
```suggestion
    /// </summary>
    public bool Add(string element)
```
А вот комментарий от своего метода никогда не отделяется